probe/main.go

```diff
@@ -89,19 +94,25 @@ func main() {
 		endpoint.NewReporter(hostID, hostName, *spyProcs),
 	}
 
-	if *dockerEnabled && runtime.GOOS == linux {
-		if err = report.AddLocalBridge(*dockerBridge); err != nil {
-			log.Fatalf("failed to get docker bridge address: %v", err)
-		}
+	// TODO provide an alternate implementation for Darwin.
```

Comment:
My understanding is that Darwin has no /proc FS.  So yes, this won't produce Process or Docker topologies on Darwin.  Note this PR doesn't change that behaviour - this comment isn't new.